perun-core/src/test/java/cz/metacentrum/perun/core/entry/AttributesManagerEntryIntegrationTest.java

```diff
@@ -770,7 +770,7 @@ public void getAllGroupAttributesStartWithNameWithoutNullValue() throws Exceptio
 			attributesManager.setAttribute(sess, group, a);
 		}
 
-		List<Attribute> retAttr = attributesManager.getAllAttributesStartWithNameWithoutNullValue(sess, group, AttributesManager.NS_GROUP_ATTR_OPT + ":group_test_uniqueattribute");
+		List<Attribute> retAttr = attributesManager.getAllAttributesStartWithNameWithoutNullValue(sess, group, AttributesManager.NS_GROUP_ATTR_OPT + ":specialNamespace");
```

Comment:
Special namespace is not good name. Please use just "-" instead of "_".
More information for you in TASK 2483 on Redmine.